e2e/playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig } from '@playwright/test';
+import { AzureReporterOptions } from '@alex_neo/playwright-azure-reporter/dist/playwright-azure-reporter';
+import path from 'path';
+const envPath = path.resolve(__dirname, '.env');
+import dotenv from 'dotenv';
+dotenv.config({ path: envPath });
+
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: 2,
+  /* Opt out of parallel tests on CI. */
+  /* Reporter to use. See https://playwright.devdocs/test-reporters */
+  reporter: [
+    ['list'],
+    [
+      '@alex_neo/playwright-azure-reporter',
+      {
+        orgUrl: 'https://dev.azure.com/redcrossnl',
```

Comment:
This value would also be a good ENV-variable. To keep all _really specific_ values out of the (open-source) code.

Reply:
Done!